### PR TITLE
airodump-ng: add page

### PR DESCRIPTION
**Pull request by crixodia**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by marchersimon** [comment]
Hey @crixodia,
we already have a `airmon-ng` page in the common directory. Do you want to improve this page instead?

**Comment by crixodia** [comment]
Sure, I'm on it

**Comment by crixodia** [comment]
Hi, the current airmon-ng page is almost the same. So, I've added airodump-ng instead as a new page.

File: pages/common/airodump-ng.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# airodump-ng
+
+> Capture packets and display information about wireless networks.
+> Part of `aircrack-ng`.
```
Comment (marchersimon) [inline pages/common/airodump-ng.md:4]: Do you think this line should also be added to `airmon-ng`?

File: pages/common/airodump-ng.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+> More information: <https://www.aircrack-ng.org/doku.php?id=airodump-ng>.
+
+- Capture packets and display information about a wireless network:
+
+`sudo airodump-ng {{interface}}`
+
+- Capture packets and display information about a wireless network, saving the data to a file:
+
+`sudo airodump-ng --channel {{channel}} --write {{filename}} --bssid {{mac}} {{interface}}`
```
Comment (marchersimon) [inline pages/common/airodump-ng.md:13]: From the differences of the descriptions I would expect the commands to only differ by the `--write` flag. What does `--channel` and `--bssid` do and shouldn't they be introduced by a separate example?

File: pages/common/airodump-ng.md (hunk 1 above)
Comment (crixodia) [reply pages/common/airodump-ng.md:4]: Hi, well I think it should be added based on the other pages where it is specified as that. Just let me know if I can add that change to this PR.

Comment (marchersimon) [reply pages/common/airodump-ng.md:4]: Sure, you can add it.

File: pages/common/airodump-ng.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+> Capture packets and display information about wireless networks.
+> Part of `aircrack-ng`.
+> More information: <https://www.aircrack-ng.org/doku.php?id=airodump-ng>.
+
+- Capture packets and display information about a wireless network:
+
+`sudo airodump-ng {{interface}}`
+
+- Capture packets and display information about a wireless given the mac address and channel, and save the output to a file:
```
Comment (marchersimon) [inline pages/common/airodump-ng.md:11]:
```suggestion
- Capture packets and display information about a wireless given the MAC address and channel, and save the output to a file:
```

File: pages/common/airodump-ng.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+`sudo airodump-ng --channel {{channel}} --write {{filename}} --bssid {{mac}} {{interface}}`
```
Comment (marchersimon) [inline pages/common/airodump-ng.md:13]:
```suggestion
`sudo airodump-ng --channel {{channel}} --write {{path/to/file}} --bssid {{mac}} {{interface}}`
```

File: pages/common/airodump-ng.md (hunk 3 above)
Comment (marchersimon) [reply pages/common/airodump-ng.md:11]: @crixodia could you please also apply this suggestion?

**Review by navarroaxel — APPROVED** [review]

File: pages/common/airodump-ng.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+> Capture packets and display information about wireless networks.
+> Part of `aircrack-ng`.
+> More information: <https://www.aircrack-ng.org/doku.php?id=airodump-ng>.
+
+- Capture packets and display information about a wireless network:
+
+`sudo airodump-ng {{interface}}`
+
+- Capture packets and display information about a wireless given the MAC address and channel, and save the output to a file:
```
Comment (kbdharun) [inline pages/common/airodump-ng.md:11]:
```suggestion
- Capture packets and display information about a wireless network given the MAC address and channel, and save the output to a file:
```

**Review by kbdharun — APPROVED** [review]
LGTM, after the above suggestion.

**Pull request merged**